kbfsgit/runner.go

```diff
@@ -1750,6 +1750,8 @@ func (r *runner) processCommands(ctx context.Context) (err error) {
 	reader := bufio.NewReader(r.input)
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
+	// Allow the creation of .kbfs_git within KBFS.
```

Comment:
So the autogit checkout will be at e.g. `/keybase/private/jzila/.kbfs_autogit/team/keybase.devops/.kbfs_git/secrets.git`, where this key allows that `.kbfs_git` to be created? Why not just skip the `.kbfs_git` and go straight to the git repos, since that can be implicit?

Reply:
No, the `.kbfs_git` is not present in the autogit path.  The line here is adding permission for the kbfsgit remote helper, not for anything to do with autogit.  It's needed because I changed the way the overrides work, so it can't be implicit here with the mode anymore, like it used to be.